Weather

```diff
@@ -1,2 +1,2 @@
-27.12. The weather is cloudy. The temperature is 11 degrees in New York.
```

Comment:
Missing word "city" after New York to specify whether it is New York State or New York city

Reply:
I will not add such bullshit